<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>closeSubwindowDialog</class>
 <widget class="QDialog" name="closeSubwindowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Close window</string>
  </property>
  <property name="locale">
   <locale language="English" country="Australia"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>671</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="instructionLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="dontaskCheckBox">
      <property name="text">
       <string>Don't ask a&amp;gain</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>C&amp;ancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nosaveButton">
        <property name="text">
         <string>&amp;Don't save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>&amp;Save</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dontaskCheckBox</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>nosaveButton</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>closeSubwindowDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nosaveButton</sender>
   <signal>clicked()</signal>
   <receiver>closeSubwindowDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>closeSubwindowDialog</receiver>
   <slot>saveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveData()</slot>
 </slots>
</ui>
